src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+@import "../utils/variables.scss";
```

Comment:
Just import it in main file before other blocks, no need to import it in every file